research/gaia/gremlin/gs_gremlin/src/graph_proxy/global_storage.rs

```diff
@@ -446,17 +446,10 @@ fn build_partition_label_vertex_ids(
 /// Transform type of ids to PartitionVertexIds as required by graphscope store,
 /// which consists of (PartitionId,Vec<VertexId>)
 fn build_partition_vertex_ids(
-    ids: &[ID],
+    id: &ID,
     graph_partition_manager: Arc<dyn GraphPartitionManager>,
 ) -> Vec<PartitionVertexIds> {
```

Comment:
Why not `get_partition_vertex_id() -> PartitionVertexId`. Also, id is better using `id: ID` without reference.

Reply:
How about the other function with label_ids ?